README.md

```diff
@@ -97,9 +99,11 @@ Visualisation:
 | [matplotlib][] | Main Python plotting library.  Start from [matplotlib gallery][] then adapt to your application. |
 | [seaborn][] | Easier to use plotting library with some statistical routines. Builds on matplotlib, but annoying to customize. |
 | [vegascope][] | View Vega/Vega-Lite plots in your web browser from local or remote Python processes. |
+| [wandb][] | Weights & Biases makes it easy to track your experiments, manage & version your data, and collaborate with your team so you can focus on building the best models. |
```

Comment:
hmm, this is specific to ML, I wouldn't say it necessarily fits in this category. Maybe in the ML category?

Reply:
Not necessarily, wandb Tables are a powerful tool that can be used to centralize exploratory data analysis. wandb can be used to visualize and query interesting rows from your datasets. Group, sort, filter, generate calculated columns, and create charts from tabular data.

Reply:
Addressed in abdc30197dd81f0604455a61299cff91fc69deec